<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterCamionGuiClass</class>
 <widget class="QDialog" name="AjouterCamionGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjouterCamionGui</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de série</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Immatriculation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Poids</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre d'essieux</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNiv">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leImma">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePoids">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEssieux">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOkay">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOkay</sender>
   <signal>clicked()</signal>
   <receiver>AjouterCamionGuiClass</receiver>
   <slot>validerCamion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerCamion()</slot>
 </slots>
</ui>
